web/packages/teleport/src/Audit/fixtures/index.ts

```diff
@@ -4403,6 +4403,48 @@ export const events = [
     app_name: 'mcp-everything',
     error: 'Invalid token: audience mismatch',
   },
+  {
+    'addr.remote': '127.0.0.1:62460',
+    cluster_name: 'teleport.dev',
+    code: 'TVNET001I',
+    ei: 0,
+    event: 'vnet_config.create',
```

Comment:
I wonder if we should use `vnet.config_create` as the event name instead. This way if we create more audit events for VNet they can all start with `vnet.`